openpype/hosts/fusion/deploy/Config/openpype_menu.fu

```diff
@@ -41,6 +41,176 @@
             },
         },
     },
+    Action
+    {
+        ID = "OpenPype_Workfiles",
+        Category = "OpenPype",
+        Name = "Workfiles...",
+
+        Targets =
+        {
+            Composition =
+            {
+                Execute = _Lua [=[
+                    data = {menu_item = "workfiles", uuid = bmd.getappuuid()}
+                    local scriptPath = app:MapPath("OpenPype:MenuScripts/openpype_menu_launch.py")
+                    target:RunScript(scriptPath, { data = data })
```

Comment:
I'm wondering if at some point we can make this so that the socket emission doesn't go through Python but straight through lua. That'd require any python dependencies to be removed completely from Fusion plus I've always found that running Python scripts compared to Lua run relatively slowly (with some delay). Much less of a delay than when also importing PySide of course, but still - I wonder if it'd be a worthwhile optimization.

Reply:
That was my hope but to use sockets in Lua you have to install an external library through LuaRocks. Maybe there are other ways to solve it but I didn't find any other way.